bug_relationship_add.php

```diff
@@ -54,7 +54,7 @@
 		'query' => array( 'issue_id' => $f_src_bug_id ),
 		'payload' => array(
 			'type' => array( 'id' => $f_rel_type ),
-			'issue' => array( 'id' => $t_dest_bug_id )
+			'issue' => array( 'id' => trim( $t_dest_bug_id ) )
```

Comment:
Since this is supposed to be a bug id (i.e. a number), it would be better to typecast to `int` IMO